src/test/java/seedu/address/logic/commands/ViewCommandTest.java

```diff
@@ -23,14 +23,14 @@ public class ViewCommandTest {
 
     @BeforeEach
     public void setUp() {
-        model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+        model = new ModelManager(getTypicalClinicConnectSystem(), new UserPrefs());
         unregisteredNric = new Nric("S1234567A");
     }
 
     @Test
     public void execute_validPatient_success() {
         Patient patientToView = model.getFilteredPatientList().get(INDEX_FIRST_PATIENT.getZeroBased());
-        ModelManager expectedModel = new ModelManager(model.getAddressBook(), new UserPrefs());
+        ModelManager expectedModel = new ModelManager(model.getClinicConnectSystem(), new UserPrefs());
         CommandResult expectedCommandResult = new CommandResult(String.format(ViewCommand.MESSAGE_VIEW_SUCCESS,
                 patientToView.getName()), null, false, patientToView, true,
                 false);
```

Comment:
Maybe you could just put this in one line

Reply:
Done